### ISSUE TITLE
Admin Theme - Widgets - Sort Icons Not Correct

### ISSUE THREAD

**Issue by rtpHarry** [OrchardCMS/Orchard#7269]
The sort icons on the widgets admin index page aren't correct:

![image](https://cloud.githubusercontent.com/assets/1038062/19213959/9fe5cf1a-8d70-11e6-9a93-f97a4f0eb0b8.png)

This is because:

  - the "switchable" javascript doesn't support specifying custom icons
  - perhaps there isn't an appropriate icon in fa to cover it - may need a custom svg creating

Old icons:

![image](https://cloud.githubusercontent.com/assets/1038062/19213970/ddbfbc7e-8d70-11e6-9ef7-5bd3a117ff71.png)

**Comment by Skrypt** [comment]
I think in this view it's used differently than what it should do. The thing is that the switchable is meant to display in grid view or list view. Here we are using to hide zones that don't have any widgets. So maybe we should have a version of that script that would be named "hideable". Or just extend the current script to be able to have custom icons.

**Comment by rtpHarry** [comment]
Yeah, it could be easily extended to accept optional params that will give alternative classes for the icons.

**Comment by Skrypt** [comment]
We still need to fix the problem with the widget order controls. They don't work.

**Comment by rtpHarry** [comment]
@Skrypt are you sure you have ruled that out? For me they were definitely not working the first time through, then when I came back to do the widgets issues they were working and then since then it seems you have seen them not working? Could be some edge case that is cropping up?

The funny thing is when I did test them the 2nd time I did experience a kind of bug but couldn't recreate it. I had been testing `widget create` in a different branch but the same db so I had about 6 recentblogpost widgets and a blogarchives widget. When I pressed the first up button the blogarchives widget jumped from the end of a list of 6 recentblogpost widgets up to the top of it. All subsequent moving around was normal even when I put it back down to its original spot and pressed up.

**Comment by Skrypt** [comment]
I've seen them not working because I had only one widget in a zone. I forgot we could'nt move them from one zone to an other by using those arrows. So, I created 2 widgets in the same zone and it worked. Apart from that I looked at that view code and nothing changed really for the new admin theme. So, if you got a bug on that it is more likely to not be only related to the new admin theme.

I think a scenario that could have happened is that you created different widgets in different zones. Their display order would have been = 1. Then you probably merged those widgets in the same zone and they had the same display order. So it ordered them by name simply. And when you clicked on the last one it has set it's display order to 0. Which made it move on top. Somehow it works like it should.

I think, if that's what's happening, then we should just rethink this by using drag and drop of widgets inside zones intead.
